infrastructure/stacks/development-pipeline/codebuild.tf

```diff
@@ -143,10 +143,9 @@ resource "aws_codebuild_project" "di_build" {
   }
 }
 
-resource "aws_codebuild_project" "di_deploy_dev" {
-  for_each       = local.deploy_envs
-  name           = "${var.project_id}-${var.environment}-deploy-${each.key}-stage"
-  description    = "Deploy to the ${each.key} environment"
+resource "aws_codebuild_project" "di_deploy" {
+  name           = "${var.project_id}-${var.environment}-deploy-stage"
```

Comment:
Is this line should be  "${var.project_id}-${var.environment}-deploy-**dev**-stage" ?